Add davidjumani and rohityadavcloud to kubernetes org

davidjumani opened the pull request:
Adds @davidjumani and @rohityadavcloud as a member of the kubernetes org
- Required for https://github.com/kubernetes/k8s.io/pull/3792#issuecomment-1143127185
- Fixes https://github.com/kubernetes/org/issues/3488

mrbobbytables reviewed (commented):
@davidjumani we have very specific formatting for how we do commit messages to add/remove people the org. We can move forward with this, but please update your commit message to:
`Add davidjumani to kubernetes org`

davidjumani commented:
Thanks @mrbobbytables I've changed the commit message
Also added @rohityadavcloud membership. Have not raised a separate issue for his membership based on your comment on the original issue

ameukam commented:
/ok-to-test

mrbobbytables reviewed (commented):
Thanks!
/lgtm
/approve

palnabarun commented:
/area github-membership
